webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -57,6 +57,15 @@
         <param-value>false</param-value>
     </context-param>
 
+	<context-param>
+		<param-name>Owasp.CsrfGuard.Config</param-name>
+		<param-value>/WEB-INF/csrfguard/Owasp.CsrfGuard.properties</param-value>
```

Comment:
I some how feel that implementers may need to override or customise the contents of this file. So what do you think of keeping it in the application data folder? Just like we have the runtime properties file.